roles/mongodb-common/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
     - mongodb-org-shell={{ mongodb.version }}
 
 - name: install mongodb config files
-  template: src=etc/mongodb/mongod.conf dest=/etc/mongod.conf owner=mongodb mode=0640
+  template: src=etc/mongodb/mongod.conf dest=/etc/mongod.conf owner=root group=mongodb mode=0640
```

Comment:
Well, it's setting the thing serverspec is looking for, so I'm okay with it being in here.